src/wfs.c

```diff
@@ -621,6 +873,78 @@ void wfs_apply_symmetry(wfs_t * sim)
   herm_destroy(hermz);
 }
 
+/**
+ * wfs_apply_symmetry2D:
+ * Make the complex fields have Hermitian symmetry.
+ */
+void wfs_apply_symmetry2D(wfs_t * sim)
```

Comment:
If we pass the dft as a parameter, we can call this separately for each component and eliminate the duplication:
```suggestion
void wfs_apply_symmetry2D(wfs_t * sim, dft_t * dft)
```